### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by thorium-zero**
image
image
image

### CODE REVIEW

File: taxi/admin.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    search_fields = ["model"]
+    list_display = ["model", "manufacturer"]
+    list_filter = ["manufacturer"]
+
+
+class ManufacturerAdmin(admin.ModelAdmin):
+    list_display = ["name", "country"]
+
+
+admin.site.register(Manufacturer, ManufacturerAdmin)
```
Comment (LiudmylaKulinchenko) [inline taxi/admin.py:33]: Better to use `@admin.register` decorator

File: taxi/models.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
```
Comment (LiudmylaKulinchenko) [inline taxi/models.py:20]: No need to reassign, those are default values

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: taxi/models.py (hunk 2 above)
Comment (thorium-zero) [reply taxi/models.py:20]: does not work without it

Comment (LiudmylaKulinchenko) [reply taxi/models.py:20]: Yess, it will be `user` by default, because of AbstractUser, sorry

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 🍬

**Pull request closed without merging**